cluster/addons/dns/README.md

```diff
@@ -13,6 +13,12 @@ crashes or scheduling changes).  This maps well to DNS, which has a long
 history of clients that, on purpose or on accident, do not respect DNS TTLs
 (see previous remark about Pod IPs changing).
 
+## DNS Name format for Services
+Services currently get a service name with the format my-svc.my-namespace.cluster.local
```

Comment:
nit: I would state the new format first and then mention the old format and mark it deprecated.